examples/octahedral cnn.py

```diff
@@ -0,0 +1,34 @@
+import torch
```

Comment:
There is a whitespace in the filename. It would be great if you remove it or replace it by an underscore.